recipe/meta.yaml

```diff
@@ -285,13 +286,15 @@ outputs:
       host:
         # Require python so that the CONDA_PY gets populated
         - python
+        # Keep the cuda version here since it helps package solvers
+        # decide on the cuda variant
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
```

Comment:
This is the correct place. If it was me who added it in `run`, it was a mistake.

Reply:
Thank you!

ok i'm going to upload the completed cuda 12.0 builds. I don't think others need a rebuild.